concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -261,6 +261,21 @@ resources:
       versioned_file: concourse_id_rsa
       initial_version: "-"
 
+  - name: cf-vars-store
+    type: s3-iam
+    source:
+      bucket: ((state_bucket))
+      region_name: ((aws_region))
+      versioned_file: cf-vars-store.yml
```

Comment:
This name is now confusing, because it is a variables store used by Bosh, and therefore may have nothing to do with Cloud Foundry (`cf-`) (we use this pipeline to deploy our CI environment as well as Cloud Foundry).

I am aware this has already been merged ;)

Reply:
This actually does have to do with CF. It's pulling in the CF vars store so that we can extract the existing bosh-DNS certs from it as part of migrating bosh-DNS here.  Once the migration has settled, there will be a follow-up to remove the usage of this.